packages/near-membrane-base/src/membrane.ts

```diff
@@ -1212,10 +1141,25 @@ export function init(
 
     // exporting callable hooks for a foreign realm
     foreignCallableHooksCallback(
+        // globalThisPointer
+        // When crossing, should be mapped to the foreign globalThis
+        createPointer(globalThis),
+        // getSelectedTarget
         getSelectedTarget,
         // getTransferableValue
         getTransferableValue,
-        // evaluate
+        // callableGetPropertyValuePointer: this callable function allows the foreign
+        // realm to access a linkable pointer for a property value. In
+        // order to do that, the foreign side must provide a pointer and a key
+        // access the value in order to produce a pointer
+        (targetPointer: Pointer, key: PropertyKey) => {
```

Comment:
this new callable allows the marshal of the environment to access a property value of a provided pointer, and the result is a pointer. This is a new capability, because until now, the environment object didn't know anything about pointers, was an implementation detail of the membrane that now leaks into a higher level. Now, there is no much the environment can do with a pointer other than call back into one of the sides of the membrane with it, so, it seems safe.